README.md

```diff
@@ -5,7 +5,7 @@ This command line application provides several ways of interacting with the [Arc
 ## Installation
 Clone repo and install using [pipenv](https://github.com/pypa/pipenv):
 ```bash
-pipenv install -e .
+pipenv run pip install -e .
```

Comment:
This is an edit I made a while back but I hadn't had any other PRs on this repo until now. My virtual envs had blown up and after clearing stuff out, I was able to successfully reinstall it with `pipenv run pip install -e .` and I had tried `pipenv install` earlier to no avail. I'm going to revisit the installation issue as this gets closer to an initial release and I'm completing the documentation

Reply:
Using `pipenv run asaps` is definitely the better way to go. Using `pipenv run pip install -e .` will make the `asaps` command available to the virtualenv, but this will not add the local module to the Pipfile (which you do not want). If you were to run--and you should, on occasion--`pipenv clean` then the local module would be uninstalled and you'd have to rerun your `pip install`.

Reply:
Updated to @hakbailey's suggestion after a chat with her and @gravesm